lambdaclass/mirra_backend
# [GH-720] Add Game Configuration

## Motivation
Add game configuration schema that represents the key in "game" in `game_config.json`

Closes #720 

## Summary of changes

- Added MVC for Game Configuration
    - Added `GameConfiguration` schema in GameBackend
    - Created a provisionary context `Configuration` in GameBackend
    - Added a new controller `ConfigurationController` in `GameBackend` that will be in charge of handling the api requests for the configurations
    - Added some seeds for game_backend for the new game configuration
- Now the game is using the game configuration from the configurator and not from the `game` key from `config.json`
- Removed `CharacterController` from the backend since it is no longer used
- Removed some default css styles that could make the application look weird

## How to test it?
- Go to `localhost:4100/game_configurations` and see the list of game configurations we have. Also, check that you can create and edit them without any problems.
- Play a game and that everything is running without crashes or problems

## Checklist
